packages/cli/generators/relation/base-relation.generator.js

```diff
@@ -98,23 +98,17 @@ module.exports = class BaseRelationGenerator extends ArtifactGenerator {
       type: this._getRepositoryRelationPropertyType(),
     };
 
-    if (relationUtils.doesPropertyExist(classDeclaration, property.name)) {
-      throw new Error(
-        'property ' + property.name + ' already exist in the repository.',
-      );
-    } else {
-      relationUtils.addProperty(classDeclaration, property);
-    }
+    // already chcked the existence of property before
```

Comment:
(sorry I am not very familiar with the relation cli which results in many questions ❔ ❔ ) Hmm, where is it checked? did a quick search of code before it but didn't find any clue 🙈

Reply:
Here the property is the relation name, which has been checked in [`index file`](https://github.com/strongloop/loopback-next/pull/4373/files#diff-d9d0cb4d78404ced50ce6c783a09ccbaR488). Sorry the PR is big, I should've separated them.